contracts/Registry/TokenCuratedRegistry.sol

```diff
@@ -34,15 +34,22 @@ contract TokenCuratedRegistry is StakedRegistry {
     challengeFactory = _challengeFactory;
   }
 
-  function add(bytes32 data) public returns (bytes32) {
+  function apply(bytes32 data) public returns (bytes32) {
     bytes32 id = keccak256(data);
     itemsCurationData[id] = ItemCurationData(now + applicationPeriod, false, address(0), false);
-    return super.add(data);
+    return addStakedItem(data);
   }
 
-  function remove(bytes32 id) public itemNotLockedInChallenge(id) {
+  function addStakedItem(bytes32 data) public returns (bytes32) {
```

Comment:
maybe we should move the application phase require to `apply` and remove this function?

Reply:
Not sure I'm understanding how that would work. I'm trying to prevent people from calling addStakedItem, without having applied. Then things would get onto the registry without any curationdata